### PR TITLE
server and docs folder created successfully

### PR DESCRIPTION
**Pull request by KessNelly**
http server created
docs folder created with a markdown file containing documentation attached

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const http = require('http');
```
Comment (OVECJOE) [inline index.js]: Please format your code...

Use consistent indentation.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Project Title
```
Comment (OVECJOE) [inline docs/README.md]: Nope.

The `README.md` you created is fine, but it should be outside the `docs` folder.

The `docs` folder is for documenting your tasks. For example, you can create a file called `task_1.md` in the `docs` folder and move the content in **"the process"** section of the README into it.

Inside the `task_1.md` file, you can then add other sections like the dependencies you installed for the task, the timeline (dated), the difficulty level, and so on.

**Review by OVECJOE — CHANGES_REQUESTED** [review]
Also, rename `publics` folder to `public` folder.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Project Title
+Authiclay
+
+## Project Description
+A small sized authenticator system using pure NodeJs, that acts as a 2 Factor authentication for web applications.
+
+## Prerequisites
+Node Js
+
+## The Process
```
Comment (OVECJOE) [inline README.md:10]: You don't need this process, yet. You can remove the content in this section.

**Review by OVECJOE — CHANGES_REQUESTED** [review]

**Review by OVECJOE — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (KessNelly) [reply README.md:10]: Done.

**Pull request merged**